src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -62,6 +63,11 @@ public CommandResult execute(String commandText) throws CommandException, ParseE
         return model.getFilteredPersonList();
     }
 
+    @Override
+    public Transcript getTranscript() {
+        return (Transcript) model.getTranscript();
```

Comment:
And thus this should not be casted.

Reply:
Yup. This was because in my prototype implementation I put the getters in Transcript.  Will update once #106 is merged.